servers/src/main/java/tachyon/master/file/journal/InodeDirectoryEntry.java

```diff
@@ -35,8 +35,10 @@ public InodeDirectoryEntry(@JsonProperty("creationTimeMs") long creationTimeMs,
       @JsonProperty("pinned") boolean pinned,
       @JsonProperty("lastModificationTimeMs") long lastModificationTimeMs,
       @JsonProperty("childrenIds") Set<Long> childrenIds,
-      @JsonProperty("permissionStatus") PermissionStatus ps) {
-    super(creationTimeMs, id, name, parentId, persisted, pinned, lastModificationTimeMs, ps);
+      @JsonProperty("username") String username, @JsonProperty("groupname") String groupname,
```

Comment:
I have an outstanding PR that fixes that ... if my PR is merged first, please resolve my comment, otherwise I will take care of this when merging conflicts